MdeModulePkg/Universal/CapsulePei/CapsuleX64.inf

```diff
@@ -50,3 +50,11 @@
 
 [UserExtensions.TianoCore."ExtraFiles"]
   CapsuleX64Extra.uni
+
+## MS_CHANGE_211478
+## MSChange [BEGIN] - This is necessary to avoid the alignment change introduced with the Memory Attributes Table

```

Comment:
This seems different than the current PR scope which is "Fixing compilation warnings/errors from GCC on Linux in MU Code".

Reply:
In general, the PR title and body don't seem to match. I'll come back later.

Reply:
We can drop this, it got updated by adding NX support: https://github.com/microsoft/mu_basecore/commit/ac74e7e44de48181e2ce33062c48d794a3d53df8. I am carrying a version of that commit here: https://github.com/microsoft/mu_basecore/pull/912/commits/4c6c82e3b8c089fa88c56de566a0d6e947f36e18.

I'm not sure the history here, it doesn't really feel related to memory protections, but I don't know why it is separate. I'm also fine to pull it out of the memory protections PR and put it here, just taking the updated lines.

Reply:
I slimmed down the PR considerably and updated the PR title and description to match the changes.